t/python/t0005-kvs.py

```diff
@@ -334,6 +335,50 @@ def test_36_put_symlink(self):
         flux.kvs.commit(self.f)
         self.assertFalse(flux.kvs.exists(self.f, "txn_symlink"))
 
+    def bad_input(self, func, *args):
+        got_einval = False
+        try:
+            func(*args)
+        except EnvironmentError as e:
+            if e.errno == errno.EINVAL:
+                got_einval = True
+        self.assertEqual(got_einval, True)
```

Comment:
I think the `unittest` way to do this is:
```python
with self.assertRaises(OSError) as ctx:
    func(*args)
self.assertEqual(ctx.exception.errno, errno.EINVAL)
```

See https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises

Also, since 3.3 I think `OSError` is preferred over `EnvironmentError`.

Reply:
ahhh this pattern is used elsewhere, will try and clean up everywhere